### PR TITLE
core: rename "network" policy to "crosscore-readonly"

### PR DESCRIPTION
**Pull request by tessr**

### CODE REVIEW

**Comment by tessr** [comment]
PTAL

File: core/authz.go [hunk 1]
```diff
@@ -40,11 +40,11 @@ var policyByRoute = map[string][]string{
...
 
-	networkRPCPrefix + "get-block":         {"network"},
-	networkRPCPrefix + "get-snapshot-info": {"network"},
-	networkRPCPrefix + "get-snapshot":      {"network"},
-	networkRPCPrefix + "signer/sign-block": {"network"},
-	networkRPCPrefix + "block-height":      {"network"},
+	networkRPCPrefix + "get-block":         {"crosscore-readonly"},
+	networkRPCPrefix + "get-snapshot-info": {"crosscore-readonly"},
+	networkRPCPrefix + "get-snapshot":      {"crosscore-readonly"},
+	networkRPCPrefix + "signer/sign-block": {"crosscore-readonly"},
```
Comment (jbowens) [inline core/authz.go]: maybe make this crosscore-signblock now since it's not really readonly?

Comment (tessr) [inline core/authz.go]: I'll add a todo, since `crosscore-signblock` doesn't exist yet.

**Comment by jbowens** [comment]
I came here for the A+ gifs.

**Pull request merged**